### ISSUE TITLE
The rOpenSci stack

### ISSUE THREAD

**Issue by karthik** [ropensci/roweb#162]
blog post idea. Just like Max's how the Dat team works (https://github.com/maxogden/async-team/blob/master/readme.md) perhaps we could do one for rOpenSci too, with more details on our stack.  If what we're doing is not that unique, we could skip it or wait till a later time where we  are in a better spot to do this.

**Comment by sckott** [comment]
good idea. even if not entirely unique, could still be interesting to many, and surely will be unique in some ways, and could be way for us to reflect on the way we work and the tools we use

**Comment by karthik** [comment]
Cool, I'll start jotting down some notes in drafts.

**Comment by cboettig** [comment]
:+1:

**Comment by karthik** [comment]
So I started writing down an outline.


# The rOpenSci workflow

The rOpenSci project like many other open source teams is a highly distributed and asynchronous operation. While more of the core leadership also wear academic hats from time to time, our workflow is quite non-traditional and we thought it might be worth sharing.

1. GitHub and GitHub issues  
    The biggest part of our tool chain. Discuss how the lion's share of our repos are public although we use one repo for private discussions, grant applications and other documents that are not ready for public release.
2. Less email  
    discuss our use of slack, and limiting email as much as necessary.
3. Community engagement  
    Moving support discussion out of email as much as possible to discourse. But we still heavily engage with our community using a support queue (Help Scout), live chat (Zopim). General discussion questions that are worth immediately sharing with public go into discource.
4. Docker  & Travis  
    Testing daily builds, other infrastructure, Training.
5. Jekyll   
    How much of our site operates. All files with R code are parsed from Rmd to md before becoming html templated to our blog's style. We also use smaller Jekyll sites to send out announcements via Mail Chimp, and also our leadership meeting minutes via rss.

Did I miss anything else worth discussing? I can get this drafted up shortly (as you can tell I am procrastinating from doing more important things).
